src/chapters/data-manipulation/data-manipulation.rst

```diff
@@ -1,2 +1,162 @@
 Data Manipulation
-=================
\ No newline at end of file
+=================
+
+.. _readingDataManipulation:
+
+Readings
+--------
+
+The readings and exercises are a little different for this chapter.  Both have been combined into 2 
+jupyter notebooks and the link can be found :ref:`here<dataManipulationNotebook>`. 
```

Comment:
```suggestion
~~jupyter notebooks and the link can be found :ref:`here<dataManipulationNotebook>`.~~
```

Reply:
(if you take the suggestion in line 9, you shouldn't need line 10)

Reply:
I reworked this page and the exercises page.  Thoughts on flow now?